assets/services.json

```diff
@@ -13,81 +13,297 @@
+        "||yamaxun.cn^",
+        "||yamaxun.com^",
+        "||yamaxun^",
+        "||z.cn^",
+        "||zappos^",
+        "/.+\\.awsdns-[0-9][0-9]\\.co\\.uk$/",
+        "/.+\\.awsdns-[0-9][0-9]\\.com$/",
+        "/.+\\.awsdns-[0-9][0-9]\\.net$/",
+        "/.+\\.awsdns-[0-9][0-9]\\.org$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.biz$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.com$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.net$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.top$/",
+        "/.+\\.awsdns-cn-[0-9][a-e0-9]\\.cn$/",
+        "/^images-cn-8\\.ssl-images-amazon\\.com$/",
+        "/^images-cn\\.ssl-images-amazon\\.com$/"
       ],
       "icon_svg": "<svg xmlns=\"http://www.w3.org/2000/svg\" fill=\"currentColor\" viewBox=\"0 0 32 32\"><path d=\"M16.2,4c-3.3,0-6.9,1.2-7.7,5.3C8.4,9.7,8.7,10,9,10l3.3,0.3c0.3,0,0.6-0.3,0.6-0.6c0.3-1.4,1.5-2.1,2.8-2.1c0.7,0,1.5,0.3,1.9,0.9c0.5,0.7,0.4,1.7,0.4,2.5v0.5c-2,0.2-4.6,0.4-6.5,1.2c-2.2,0.9-3.7,2.8-3.7,5.7c0,3.6,2.3,5.4,5.2,5.4c2.5,0,3.8-0.6,5.7-2.5c0.6,0.9,0.9,1.4,2,2.3c0.3,0.1,0.6,0.1,0.8-0.1v0c0.7-0.6,2-1.7,2.7-2.3c0.3-0.2,0.2-0.6,0-0.9c-0.6-0.9-1.3-1.6-1.3-3.2v-5.4c0-2.3,0.2-4.4-1.5-6C20.1,4.4,17.9,4,16.2,4z M17.1,14.3c0.3,0,0.6,0,0.9,0v0.8c0,1.3,0.1,2.5-0.6,3.7c-0.5,1-1.4,1.6-2.4,1.6c-1.3,0-2.1-1-2.1-2.5C12.9,15.2,14.9,14.5,17.1,14.3z M26.7,22.4c-0.9,0-1.9,0.2-2.7,0.8c-0.2,0.2-0.2,0.4,0.1,0.4c0.9-0.1,2.8-0.4,3.2,0.1s-0.4,2.3-0.7,3.1c-0.1,0.2,0.1,0.3,0.3,0.2c1.5-1.2,1.9-3.8,1.6-4.2C28.3,22.5,27.6,22.4,26.7,22.4z M3.7,22.8c-0.2,0-0.3,0.3-0.1,0.4c3.3,3,7.6,4.7,12.4,4.7c3.4,0,7.4-1.1,10.2-3.1c0.5-0.3,0.1-0.9-0.4-0.7c-3.1,1.3-6.4,1.9-9.5,1.9c-4.5,0-8.8-1.2-12.4-3.3C3.8,22.9,3.7,22.8,3.7,22.8z\" /></svg>"
     },
     {
       "id": "bilibili",
       "name": "Bilibili",
       "rules": [
+        "||acg.tv^",
+        "||acgvideo.com^",
+        "||animetamashi.cn^",
+        "||animetamashi.com^",
+        "||anitama.cn^",
+        "||anitama.net^",
         "||b23.tv^",
+        "||bigfun.cn^",
+        "||bili22.cn^",
+        "||bili2233.cn^",
+        "||bili23.cn^",
+        "||bili33.cn^",
+        "||biliapi.com^",
         "||biliapi.net^",
+        "||bilibili.cc^",
         "||bilibili.com^",
+        "||bilibili.net^",
+        "||bilibili.tv^",
+        "||bilibiligame.cn^",
+        "||bilibiligame.co^",
+        "||bilibiligame.net^",
+        "||bilibilipay.cn^",
+        "||bilibilipay.com^",
         "||bilicdn1.com^",
         "||bilicdn2.com^",
+        "||bilicdn3.com^",
+        "||bilicdn4.com^",
+        "||bilicdn5.com^",
+        "||biligame.co^",
         "||biligame.com^",
+        "||biligame.net^",
+        "||biligo.com^",
         "||bilivideo.cn^",
         "||bilivideo.com^",
+        "||bilivideo.net^",
         "||dreamcast.hk^",
-        "||hdslb.com^"
+        "||hdslb.com^",
+        "||hdslb.org^",
+        "||im9.com^",
+        "||maoercdn.com^",
+        "||mincdn.com^",
+        "||yo9.com^",
+        "/^upos-hz-mirrorakam\\.akamaized\\.net$/"
```

Comment:
Replace with a regular rule please

Reply:
Does Adblock-style syntax support not matching subdomains only matching the domain that I write?

Reply:
Finish